lib/engine.rb

```diff
@@ -2,28 +2,58 @@
 
 # Engine creates messeges to communicate with users and gets their inputs as well as running the game logic
 module Engine
-  def self.start
-    @hash = {}
+  def self.play(game, board)
```

Comment:
I am wondering why you chose to define your methods like
`def self.play` as opposed to `def play` etc